hack/prom-rule-ci/prom-rules-tests.yaml

```diff
@@ -639,3 +639,22 @@ tests:
             value: 103706112
           - labels: 'kubevirt_vmi_memory_used_bytes{container="virt-handler", name="vm-example-2", namespace="default", node="node-1"}'
             value: 234329980
+
+  # Excessive VMI Migrations in a period of time
+  - interval: 24h
+    input_series:
+      - series: 'kubevirt_migrate_vmi_succeeded_total{vmi="vmi-example-1"}'
+        values: "0 1 2 3 4 5 6 7 8 9 10 11 12 13"
+
+    alert_rule_test:
+      - eval_time: 24h
```

Comment:
@orenc1 I think you also need to add a check that checks the alert is not firing before and after.

Reply:
added.